src/style.css

```diff
@@ -1 +1,80 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --color-font: #000;
+  --color-active: #00acdc;
+}
+
+html {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)